packages/expo-image-picker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-image-picker",
-  "version": "14.3.0",
+  "version": "14.3.1",
```

Comment:
Revert this please. This will be done by the publish scripts.

Reply:
done